masonry/src/testing/helper_widgets.rs

```diff
@@ -105,9 +107,10 @@ pub enum Record {
     PE(PointerEvent),
     TE(TextEvent),
     AE(AccessEvent),
-    RegisterChildren,
+    AF(u64),
+    RC,
```

Comment:
Why are there no doc comments here? I can only work out what this refers to *because I have this diff open*

I'm presuming the added terseness is to decrease the size of the created snapshots. But it looks to me like there's only one snapshot that uses this event, looking at the diff?

Reply:
> But it looks to me like there's only one snapshot that uses this event, looking at the diff?

A fair number of tests have been commented out through refactors. I'd like to progressively re-add them. I *think* some of them will use this event?

> Why are there no doc comments here?

Yeah, there probably should be.

I'm not sure the terse names are really needed either. The snapshots all use the "one-line per nesting level" format anyway.